.github/workflows/label-triaged.yml

```
@@ -0,0 +1,24 @@
+name: 🤖 Issue labeler - Triaged
+on:
+  issues:
+    types:
+      - labeled
+jobs:
+  add-comment:
+    if: github.event.label.name == 'triaged: bug'  
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Add comment
+        uses: peter-evans/create-or-update-comment@v2
```

Lets use the git hash so that it doesnt automatically update if the tag is changed
```suggestion
        uses: peter-evans/create-or-update-comment@5adcb0bb0f9fb3f95ef05400558bdb3f329ee808
```